alpine-golang/alpine3.13-golang1.15/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.15-alpine3.13
+
+RUN apk update
+
+RUN apk add -U build-base
```

Comment:
This is not ideal.

At least do:

```
apk add --no-cache -U build-base
```

Reply:
Done. Force pushed. Also updated the 3.12 image for completeness' sake.